connector/spanmetricsconnector/connector.go

```diff
@@ -125,6 +129,16 @@ func newConnector(logger *zap.Logger, config component.Config, ticker *clock.Tic
 		resourceMetricsKeyAttributes[attr] = s
 	}
 
+	var lastDeltaTimestamps *simplelru.LRU[metrics.Key, pcommon.Timestamp]
+	if cfg.GetAggregationTemporality() == pmetric.AggregationTemporalityDelta {
+		lastDeltaTimestamps, err = simplelru.NewLRU[metrics.Key, pcommon.Timestamp](cfg.GetDeltaTimestampCacheSize(), func(k metrics.Key, v pcommon.Timestamp) {
```

Comment:
Can we use `cache.NewCache` instead? This will help us maintain uniformity.

Reply:
We could but `cache.NewCache` holds onto items evicted from its LRU in memory, which isn't needed like it is for the other caches. We would then have to remember to call `RemoveEvictedItems()` to discard those items from memory